internal/app/app.go

```diff
@@ -94,7 +96,21 @@ func (a *App) DomReady(ctx context.Context) {
 
 	a.config.RunMigrations()
 	a.systrayMgr.Init(ctx)
-	cfg.SelfUpdate(ctx)
+
+	su, err := selfupdate.NewSelfUpdater(&http.Client{
+		Timeout: 20 * time.Second,
+	})
+	if err != nil {
+		log.Printf("error creating self updater: %v", err)
+		return
+	}
```

Comment:
Thoughts on placing this inside the goroutine as well? That way we'll still go through with automatic startup even if selfupdater failed to initialize.